ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: base
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
```

Comment:
for what is this used?

Reply:
Development environment for baremetal

Reply:
And without that image all the e2e-metal-ipi workflows couldn't work (see [1])

[1] https://github.com/openshift/release/blob/662bbc163381c6af0f48d8488444c9f9707069d1/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-ref.yaml#L3